Documentation/design-docs/host-tracing.md

```diff
@@ -7,7 +7,7 @@ Currently (as of .NET Core 2.1) the host tracing is only written to the `stderr`
 
 ## Proposed changes
 For .NET Core 3 the proposal is to keep the existing behavior as described above and add two additional ways to capture the tracing:
-* Redirect the trace to a file instead - this would be done by setting `COREHOST_TRACE=1` and also `COREHOST_TRACEFILE=<path>` in the environment for the process.
+* Redirect the trace to append a file instead - this would be done by setting `COREHOST_TRACE=1` and also `COREHOST_TRACEFILE=<path>` in the environment for the process.
```

Comment:
Nit: I would word this differently: "Redirect the trace to a file (always appends) - ..."